ext/js/background/backend.js

```diff
@@ -609,10 +609,14 @@ export class Backend {
 
             const valid = isNoteDataValid(note);
 
+            const noteIds = isDuplicate ? duplicateNoteIds[originalIndices.indexOf(i)] : null;
+            const noteInfos = (fetchAdditionalInfo && noteIds !== null && noteIds.length > 0) ? await this._anki.notesInfo(noteIds) : [];
+
             const info = {
```

Comment:
Is it possible to make this dict typed? Not sure how this is being consumed by the caller of `_onApiGetAnkiNoteInfo`